day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,49 @@
+numbers = (1..10)
+
+def speckled_frogs(frogs)
+  frogs.each do |frog|
+    if frog == 1
+      puts "#{frog} speckled frog sat on a log"
+    else
+      puts "#{frog} speckled frogs sat on a log"
+    end
+
+    puts "eating some most delicious bugs.\nOne jumped in the pool where it's nice and cool,"
+
+    if frog == 1
+      puts "then there were no more speckled frogs!"
+    elsif frog == 2
+      puts "then there was #{frog - 1} speckled frog!"
+    else
+      puts "then there were #{frog - 1} speckled frogs!"
+    end
+  end
+end
+
+speckled_frogs(numbers)
```

Comment:
Nice work on this exercise! Glad to see you did the extension as well. One thing I noticed here is that when testing your code, the `10_speckled_frogs` rhyme printed out from 1 - 10 rather than 10 counting down to 1 as requested in the instructions.  Can you think of a way to loop through the array of numbers in the reverse?

Reply:
Thanks for feedback and the encouragement.  To loop through the array going from 10 down to 1, I would use the .reverse method when the variable is assigned.  I tried that in the code and it works.